sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -40,9 +41,10 @@ SUBROUTINE ALLOCATE_ALL()
 !
       include 'mpif.h'
 !
-      integer ierr,i,jsx,jex
+      integer ierr,jsx,jex
+      integer i,j,l,k
 ! Allocate arrays
-      allocate(u(im+1,jsta_2l:jend_2u,lm))
```

Comment:
I checked and it looked like INITPOST.F is the only initialization subroutine to use u instead of uh.  I propose we update this subroutine to use uh too and remove two 3D arrays U and V in the near future

Reply:
@HuiyaChuang-NOAA I will coordinated with GSL and add the changes in the enhancement of  VUCSH/VVCSH 0-1km above ground fields for fv3lam in the next PR.